### ISSUE TITLE
Getting "failed to find end of block"

### ISSUE THREAD

**Issue by pdecat** [katbyte/terrafmt#11]
Hi @katbyte,

I'm getting several `failed to find end of block` errors on https://github.com/claranet/terraform-provider-zabbix/ (master branch)

```sh
# for f in zabbix/*_test.go ; do ~/go/bin/terrafmt fmt $f ; done
ERRO[0000] block 1 @ zabbix/resource_zabbix_host_group_test.go#56 failed to find end of block
ERRO[0000] block 1 @ zabbix/resource_zabbix_host_test.go#63 failed to find end of block
ERRO[0000] block 1 @ zabbix/resource_zabbix_item_test.go#53 failed to find end of block
ERRO[0000] block 2 @ zabbix/resource_zabbix_item_test.go#53 failed to find end of block
ERRO[0000] block 1 @ zabbix/resource_zabbix_template_link_test.go#141 failed to find end of block
ERRO[0000] block 2 @ zabbix/resource_zabbix_template_link_test.go#141 failed to find end of block
ERRO[0000] block 3 @ zabbix/resource_zabbix_template_link_test.go#141 failed to find end of block
ERRO[0000] block 1 @ zabbix/resource_zabbix_template_test.go#151 failed to find end of block
ERRO[0000] block 2 @ zabbix/resource_zabbix_template_test.go#151 failed to find end of block
ERRO[0000] block 3 @ zabbix/resource_zabbix_template_test.go#151 failed to find end of block
ERRO[0000] block 4 @ zabbix/resource_zabbix_template_test.go#151 failed to find end of block
ERRO[0000] block 5 @ zabbix/resource_zabbix_template_test.go#151 failed to find end of block
ERRO[0000] block 6 @ zabbix/resource_zabbix_template_test.go#151 failed to find end of block
ERRO[0000] block 7 @ zabbix/resource_zabbix_template_test.go#151 failed to find end of block
ERRO[0000] block 8 @ zabbix/resource_zabbix_template_test.go#151 failed to find end of block
```

Example block triggering the error:

```go
func testAccZabbixHostGroupConfig(groupName string) string {
	return fmt.Sprintf(`
		resource "zabbix_host_group" "zabbix" {
			name = "%s"
		}`, groupName,
	)
}
```

Note: this happens with the freshly released v0.1.0 compiled with go 1.13.8 and 1.14.

**Comment by pdecat** [comment]
Oh, issue is because the closing `` `,`` is not on its own line, and at the beginning of the line.

It works with this:

```
diff --git zabbix/resource_zabbix_host_group_test.go zabbix/resource_zabbix_host_group_test.go
index 40bed36..c45c172 100644
--- zabbix/resource_zabbix_host_group_test.go
+++ zabbix/resource_zabbix_host_group_test.go
@@ -56,7 +56,8 @@ func testAccZabbixHostGroupConfig(groupName string) string {
        return fmt.Sprintf(`
                resource "zabbix_host_group" "zabbix" {
                        name = "%s"
-               }`, groupName,
+               }
+`, groupName,
        )
 }
```

**Comment by katbyte** [comment]
closed by #12
